core/storage/exploration/gae_models.py

```diff
@@ -195,7 +220,20 @@ def _trusted_commit(
             self, committer_id, commit_type, commit_message, commit_cmds):
         """Record the event to the commit log after the model commit.
 
-        Note that this overrides the superclass method.
+        Note that this extends the superclass method.
+
+        Args:
```

Comment:
Ah, sorry, I meant this whole args block, not this line in particular. I think you've already fixed it, though!